maraboupy/MarabouNetworkNNet.py

```diff
@@ -49,13 +54,111 @@ def __init__ (self, filename, normalize=False):
             weights          (list of list of lists) Outer index corresponds to layer
                                 number.
             biases           (list of lists) Outer index corresponds to layer number.
+
+         others:
+            inputVars
+            b_variables
+            f_variables
+            outputVars
+            numVars
+
+        Attributes from parent (MarabouNetwork)
+
+            numVars
+            equList = []
+            reluList = []
+            maxList = []
+            varsParticipatingInConstraints = set()
+            lowerBounds = dict()
+            upperBounds = dict()
+            inputVars = []
+            outputVars = np.array([])
         """
         super().__init__()
 
         self.normalize = normalize
+
+        if not filename:
```

Comment:
This could be reduced to self.clearNetwork()

Reply:
I have originally considered this, but I generally prefer to include initiation of all the attributes in __init__

However, if it seems preferable to you, I don't feel strongly in this particular case, I think it will be clear enough either way.

Reply:
I think it's cleaner to define the attributes in one place. It will also make sure we don't define the attributes in different ways in each place, and will make it easier to update in the future since we only define the default attributes in one place.

Reply:
Yes, I think that you are right. I will change this.